doc/source/visualizing/plots.rst

```diff
@@ -1711,10 +1732,10 @@ of ``particle_position_x`` versus ``particle_velocity_z``, with the ``particle_m
    import yt
 
    ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
-   p = yt.ParticlePlot(ds, "particle_position_x", "particle_velocity_z", ["particle_mass"])
-   p.set_unit("particle_position_x", "Mpc")
-   p.set_unit("particle_velocity_z", "km/s")
-   p.set_unit("particle_mass", "Msun")
+   p = yt.ParticlePlot(ds, ("all", "particle_position_x"), ("all", "particle_velocity_z"), [("all", "particle_mass")])
```

Comment:
For sake of simplicity + consistency with the other example (line 1748 as of this writing)
```suggestion
   p = yt.ParticlePlot(ds, ("all", "particle_position_x"), ("all", "particle_velocity_z"), ("all", "particle_mass"))
```

Reply:
OK, so, this changes what was in there before ... I suppose they do the same thing, right?

Reply:
This is what I'm assuming too